lib/pharos/phases/validate_host.rb

```diff
@@ -163,6 +166,50 @@ def read_resolvconf
           systemd_resolved_stub: check_resolvconf_systemd_resolved_stub
         )
       end
+
+      ROUTE_REGEXP = %r(^((?<type>\S+)\s+)?(?<prefix>default|[0-9./]+)(\s+via (?<via>\S+))?(\s+dev (?<dev>\S+))?(\s+proto (?<proto>\S+))?(\s+(?<options>.+))?$)
+
+      # @return [Array<Pharos::Configuration::Host::Route>]
+      def read_routes
```

Comment:
`ValidateHost` starts to get bloated.. maybe wrap these helpers to a separate class?

Reply:
I can move the parsing to `Pharos::Configuartion::Host::Route#parse`... would that be enough?

Reply:
Could also be a separate Phase.

Reply:
I would prefer to have just single validation phase and split the logic to separate classes. `Pharos::Configuartion::Host::Route#parse` fits that bill.